api/v1alpha1/hostedcluster_types.go

```diff
@@ -456,7 +459,7 @@ type APIServerNetworking struct {
 
 // NetworkType specifies the SDN provider used for cluster networking.
 //
-// +kubebuilder:validation:Enum=OpenShiftSDN;Calico
+// +kubebuilder:validation:Enum=OpenShiftSDN;Calico;OVNKubernetes;Other
```

Comment:
do we have a valid case for other today? If not I'd suggest to drop it and only add it when that use case exists.

Reply:
@enxebre without "other", it makes it hard to test other networking providers. We want some way of telling the CNO to not install a cluster networking provider. Of course `Other` would not be supported (and we can document that fact). We don't have any validation in the equivalent fields for Installer config or `Network.operator.openshift.io`